content/en/profiler/connect_traces_and_profiles.md

```diff
@@ -117,63 +106,14 @@ This capability requires `dd-trace-dotnet` version 2.30.0+.
 {{< /programming-lang >}}
 {{< programming-lang lang="php" >}}
 
-The Trace to Profiling integration is enabled by default when you [turn on profiling for your PHP service][1].
-
-Requires `dd-trace-php` version 0.71+.
-
-To enable the [timeline feature](#span-execution-timeline-view):
-- Upgrade to `dd-trace-php` version 0.98+.
-- Set the environment variable `DD_PROFILING_TIMELINE_ENABLED=1` or INI setting `datadog.profiling.timeline_enabled=1`
+The Trace to Profiling integration is enabled when you [turn on profiling for your PHP service][1] when you meet the following criteria:
+- You are on `dd-trace-php` version 0.98+
+- You set the environment variable `DD_PROFILING_TIMELINE_ENABLED=1` or INI setting `datadog.profiling.timeline_enabled=1`
```

Comment:
Since `0.98.0` timeline is default enabled, but we had the case that a customer had it disabled and was missing the view. So maybe it's good to explicitly state this here